docs/en/api/POSIX-API.md

```diff
@@ -123,13 +180,166 @@ One possibility, for example, is to edit `${ALLUXIO_HOME}/conf/alluxio-site.prop
+    </tr>
+    <tr>
+        <td>alluxio.user.client.cache.page.size</td>
+        <td>1MB</td>
+        <td>Size of each page in client-side cache. Users can try 2MB, 4MB and 8MB first in performance tunning. Note that, if using memory to cache and the page size is too big, JVM may GC frequently. If the value is too small, the cache hit ratio may be low.</td>
+    </tr>
+    <tr>
+        <td>alluxio.user.client.cache.size</td>
+        <td>512MB</td>
+        <td>The maximum size of the client-side cache.</td>
+    </tr>
+</table>
+
+An example configuration of deep learning tasks is adding the following configuration to `${ALLUXIO_HOME}/conf/alluxio-site.properties`:
+```
+alluxio.user.client.cache.enabled=true
+alluxio.user.client.cache.store.type=LOCAL
+alluxio.user.client.cache.dir=/mnt/ramdisk
+alluxio.user.client.cache.page.size=2MB
+alluxio.user.client.cache.size=2000MB
+```
+If the cache size is big, users may need to tune the JVM as well by adding the following java opts to `${ALLUXIO_HOME}/conf/alluxio-env.sh`:
+```
+ALLUXIO_FUSE_JAVA_OPTS+=" -Xmx10G -Xms10G -XX:MaxDirectMemorySize=8g"
+```
+Users are recommended to monitor the cache hit ratio (via `${ALLUXIO_HOME}/logs/fuse.log`) and JVM GC status to adjust the page size and cache size.
+  {% endcollapsible %}
+  {% collapsible Alluxio client side metadata cache %}
+Alluxio JNI-FUSE can cache file metadata locally to reduce the overhead of
+repeatedly requesting metadata of the same file from Alluxio Master.
+Enable when the workload repeatedly getting information of numerous files.
+
+<table class="table table-striped">
+    <tr>
+        <td>Configuration</td>
+        <td>Default Value</td>
+        <td>Description</td>
+    </tr>
+    <tr>
+        <td>alluxio.user.metadata.cache.enabled</td>
+        <td>false</td>
+        <td>If this is enabled, metadata of paths will be cached. The cached metadata will be evicted when it expires after alluxio.user.metadata.cache.expiration.time or the cache size is over the limit of alluxio.user.metadata.cache.max.size.</td>
+    </tr>
+    <tr>
+        <td>alluxio.user.metadata.cache.max.size</td>
+        <td>100000</td>
+        <td>Maximum number of paths with cached metadata. Only valid if the filesystem is alluxio.client.file.MetadataCachingBaseFileSystem.</td>
+    </tr>
+    <tr>
+        <td>alluxio.user.metadata.cache.expiration.time</td>
+        <td>10min</td>
+        <td>Metadata will expire and be evicted after being cached for this time period. Only valid if the filesystem is alluxio.client.file.MetadataCachingBaseFileSystem.</td>
+    </tr>
+</table>
+
+For example, a workload that repeatedly gets information of 1 million files and runs for 50 minutes can set the following configuration:
+```
+alluxio.user.metadata.cache.enabled=true
+alluxio.user.metadata.cache.max.size=1000000
+alluxio.user.metadata.cache.expiration.time=1h
+```
+The metadata size of 1 million files usually is around 25MB to 100MB.
```

Comment:
@cheyang @apc999 I mentioned in this PR that it takes about 25MB to 100MB (to be safe, some file may have more metadata)